src/main/java/core/basesyntax/vehicles/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.vehicles;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Bulldozer is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The Bulldozer is stopped");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It could be improved to something like: 'Bulldozer has stopped working.' This makes it clear which machine has stopped.